ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml

```diff
@@ -83,7 +83,7 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
-- as: aws-ipi-byo-iam-role-f14
+- as: aws-ipi-byo-iam-role-fips-regen-cert-f14
```

Comment:
because fips are not supported on arm, so no need to enable `fips`

Reply:
Fixed